README.md

```diff
@@ -30,6 +30,46 @@ $ make build-docker-image #Builds the image, will only have to be re-run if you
 $ make docker #Runs the image
 ```
 
+## Using with your own Myradio
+
+### Inital setup
+
+Next you need a api_key to allow the website to access myradio's show information,
+
+login into database with details used during setup of myradio
+
+```sql
+INSERT INTO myury.api_key (key_string, description) VALUES ('ARANDOMSTRINGOFCHARACTERS', '2016-site development api key');
+INSERT INTO myury.api_key_auth (key_string, typeid) VALUES ('ARANDOMSTRINGOFCHARACTERS', (SELECT typeid FROM l_action WHERE phpconstant = 'AUTH_APISUDO'));
+```
+
+[please choose a better key than 'ARANDOMSTRINGOFCHARACTERS']
+
+You might need add some other database columns to create shows
+
+for example:
+-   explict podcasts (to create shows)
+-   selector (expected by 2016-site/can remove this from models/index.go 2016-site)
+
+This will fix shows not loading on 2016-site when using the base myradio as,
+
+2016-site uses parts of database that aren't made on myradio creation.
+
+### finishing steps
+
+After completing all these setups:
+You can setup a reverse proxy to "https://localhost:4443" or configure ssl for https connections,
+
+And change 2016-site to use your myradio instance:
+
+In Config.toml:
+
+```
+myradio_api = "https://{hostname}/api/v2"
```

Comment:
the problem I encoured is there no easy way of running it over https which is required by myradio apache server, without reverse proxying through a domain. 
as it looks like you might to gen some of your own certificates before you can use api 
x509: cannot validate certificate for 0.0.0.0 because it doesn't contain any IP SANs
I'll see if can make some instructions on how to do this

Reply:
if you go to http://localhost:7080/myradio/ rather than https://localhost:4443, it will work over http (if you have already set up a local instance, you need to change the port in `Config:$base_url` in `src/MyRadio_Config.local.php`)